bin/pygrb/pygrb_make_summary_page

```diff
@@ -0,0 +1,1034 @@
+# Parse command line
+# =============================================================================
+
+def parse_command_line():
+
+  parser = ArgumentParser()
+
+  parser.add_argument("-g", "--grb-name", type=str, default=None,
+                      help="GRB name.")
+
+  parser.add_argument("-f", "--config-file", type=str,
+                      default=None, help="location of the run ini file" )
+
+  parser.add_argument("-i", "--ifo-tag", type=str, default=None,
+                      help="The ifo tag, H1L1 or H1L1V1 for instance")
+
+  parser.add_argument("-t", "--start-time", type=int, default=None,
+                      help="GPS time of the GRB trigger")
+
+  parser.add_argument("-r", "--ra", type=float, default=None,
+                      help="Right ascension of external trigger")
+
+  parser.add_argument("-d", "--dec", type=float, default=None,
+                      help="Declination of external trigger")
+  
+  parser.add_argument("-o", "--output-path", type=str, default=os.getcwd(),
+                      help="output directory")
+
+  parser.add_argument("-w", "--html-path", type=str, default=None,
+                      help="final web directory")
+
+  parser.add_argument("-s", "--seg_plot", type=str, default=None,
+                      help="segment plot")
+
+  parser.add_argument("-D", "--exclusion-injections", type=str, default=None,
+                      help="A comma seperated list of the detection injection"
+                           " run names." )
+
+  parser.add_argument("-T", "--tuning-injections", type=str, default=None,
+                      help="A comma seperated list of the tuning injection "
+                           "run names.")
+
+  parser.add_argument("-O", "--open-box", default=False,
+                      help ="Use to show onsource results")
+
+  parser.add_argument("-G", "--sky-grid", default=False,
+                      help="Include sky grid information")
+
+  parser.add_argument("-M", "--mass-bins", type=str, default="0.0-8.0",
+                      help="comma separated list of dash-separated pairs "
+                           "of m_low-m_high mass bins")
+
+  parser.add_argument("-n", "--ipn", default=False,
+                      help="Include ipn information")
+  parser.add_argument("-e", "--ipn-error", type=float, default=None,
+                      help="size of the IPN error box")
+
+  args = parser.parse_args()
+
+  if not args.config_file:
+    parser.error("must provide --config-file")
+
+  if not args.ifo_tag:
+    parser.error("must provide --ifo-tag")
+
+  return args
+
+# =============================================================================
+# Main function
+# =============================================================================
+
+def main(args, inifile, outdir, htmldir, ifoTag, detectionInjList=[],\
+         tuningInjList=[], open_box=False, grid=False, ipn=False,\
+         massBins = [('0','8.0')] ):
```

Comment:
Can this function be named something more informative than "main"? Also needs documentation.